crates/router/src/services/api.rs

```diff
@@ -2037,6 +2042,11 @@ fn get_hyper_loader_sdk(sdk_url: &str) -> String {
     format!("<script src=\"{sdk_url}\" onload=\"initializeSDK()\"></script>")
 }
 
+fn get_preload_link_html_template(sdk_url: &str) -> String {
+    format!("<link rel=\"preload\" href=\"https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800\" as=\"style\">\n
+            <link rel=\"preload\" href=\"{sdk_url}\" as=\"script\">")
+}
```

Comment:
Use raw strings for better readability.

```suggestion
fn get_preload_link_html_template(sdk_url: &str) -> String {
    format!(r#"<link rel="preload" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800" as="style">\n
               <link rel="preload" href="{sdk_url}" as="script">"#);
}
```